src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-container-registry/main.combined.bicep

```diff
@@ -1,4 +1,4 @@
-resource containerRegistry 'Microsoft.ContainerRegistry/registries@2019-05-01' = {
+resource containerRegistry 'Microsoft.ContainerRegistry/registries@2020-11-01-preview' = {

```

Comment:
Note, some users have strong feelings about using preview versions. For existing snippets declared with stable api versions, I think it might be safer to just bump to the latest **stable** version rather than the latest version.

Reply:
Agreed. This was not intentional. The intellisense masks the preview on the longer types. Updated to stable.